experimental/material_3_demo/lib/component_screen.dart

```diff
@@ -1541,6 +1538,60 @@ class BottomAppBars extends StatelessWidget {
   }
 }
 
+class MenuExample extends StatefulWidget {
+  const MenuExample({super.key});
+
+  @override
+  State<MenuExample> createState() => _MenuExampleState();
+}
+
+class _MenuExampleState extends State<MenuExample> {
+  @override
+  Widget build(BuildContext context) {
+    return MenuAnchor(
+      builder: (context, controller, child) {
```

Comment:
It looks like this file generally specifies argument types

Reply:
Ah I removed the types because of the lint rules:) I guess that the sample repo just has different rule from the flutter repo.
<img width="838" alt="Screenshot 2023-01-18 at 3 59 21 PM" src="https://user-images.githubusercontent.com/36861262/213323835-82b50270-4220-4dad-8125-edc9f8f1d8bb.png">

Reply:
I see, OK.